bookstore/clone.html

```diff
@@ -65,7 +70,8 @@
 
         var button = document.getElementById("mybutton");
         button.onclick = function(event) {
-          var post_json = {{ post_model | tojson(indent=2)}}
+          button.disabled=true;
```

Comment:
minor nit pick - create space around the `=` sign.

Reply:
fixed!